native/src/main/java/io/ballerina/lib/aws/redshift/nativeimpl/ClientProcessor.java

```diff
@@ -83,33 +81,26 @@ public static Object createClient(BObject client, BMap<BString, Object> clientCo
                 .setPoolProperties(poolProperties)
                 .setConnectionPool(connectionPool, globalPool);
 
-        boolean executeGKFlag = false;
-        boolean batchExecuteGKFlag = false;
-        switch (requestGeneratedKeys) {
-            case Constants.RequestGeneratedKeysValues.EXECUTE:
-                executeGKFlag = true;
-                break;
-            case Constants.RequestGeneratedKeysValues.BATCH_EXECUTE:
-                batchExecuteGKFlag = true;
-                break;
-            case Constants.RequestGeneratedKeysValues.ALL:
-                executeGKFlag = true;
-                batchExecuteGKFlag = true;
-                break;
-            default:
-                break;
-        }
 
         return io.ballerina.stdlib.sql.nativeimpl.ClientProcessor.createClient(client, sqlDatasourceParams,
-                executeGKFlag, batchExecuteGKFlag);
+                true, true);
     }
 
-    // Unable to perform a complete validation since URL differs based on the database.
     private static boolean isJdbcUrlValid(String jdbcUrl) {
         return !jdbcUrl.isEmpty() && jdbcUrl.trim().startsWith("jdbc:");
     }
 
     public static Object close(BObject client) {
         return io.ballerina.stdlib.sql.nativeimpl.ClientProcessor.close(client);
     }
+
+    private static void addSSLOptions(BMap config, BMap<BString, Object> options) {
+        BString mode = config.getStringValue(Constants.SSL.SSL_MODE);
+        if (mode.getValue().equals(Constants.SSL.SSL_MODE_DISABLED.getValue())) {
```

Comment:
Can't we do the BString comparison without getting the String value of these two?